.gitignore

```diff
@@ -31,3 +31,5 @@
 
 /config/database.yml
 .env
+
+/vendor/*
```

Comment:
We do want to keep the vendor folder actually. It's where 3rd party files that can't be installed as gems go. It doesn't get much use anymore but it's still a legit folder.